lib/config/presets/internal/default.ts

```diff
@@ -2,6 +2,57 @@ import type { Preset } from '../types';
 
 /* eslint sort-keys: ["error", "asc", {caseSensitive: false, natural: true}] */
 export const presets: Record<string, Preset> = {
+  angularTitles: {
```

Comment:
I'm not sure so many of the fields are required here?

Reply:
Actually I am not sure either.

That are mostly options, which themself are defined as set of multiple options, like

https://docs.renovatebot.com/configuration-options/#pindigest

![grafik](https://user-images.githubusercontent.com/90388/225416766-76850fdd-59c2-49b2-b713-9941d8173dc6.png)

I had the feeling to define (e.g.) `pinDigest` to be angular compatible I have to copy all properties, which are defined there.

Reply:
I thought you are only modifying commitMessage Action

Reply:
Yes, exactly. That should be the only difference to the default settings.

Can we maybe speed it a little bit up and you just tell me, what I should do?

Asking me I would just drop the lowercase-logic and let angular-style-fans set the values on their own. If thats fine, the very first commit already solved the issue...

Reply:
Revert